homeassistant/components/shelly/device_trigger.py

```diff
@@ -110,7 +110,7 @@ async def async_get_triggers(
     hass: HomeAssistant, device_id: str
 ) -> list[dict[str, Any]]:
     """List device triggers for Shelly devices."""
-    triggers: list[dict[str, Any]] = []
+    triggers: list[dict[str, str]] = []
```

Comment:
Is it not line 57 that needs adjusting instead?

Reply:
Check the change in https://github.com/home-assistant/core/pull/72139/files

Reply:
Ok, got it now, both needs to be changed

Reply:
The branch here doesn't seem up to date.

Reply:
Rebased